### ISSUE TITLE
KBD Navigation: Button To Open A New Tab Not In Tab Order If Moving Forward

### ISSUE THREAD

**Issue by MatejGolian** [onionshare/onionshare#1595]
So, thanks to the development release build I was finally able to test the new accessibility improvements for myself. 👍
As far as I can tell so far, it's like it should be. My screen reader announces the new button labels correctly and the keyboard shortcuts for opening and closing tabs work fine as well.
I did find one minor issue though. It's not a big deal by any means, but I thought I'd bring it to your attention.
When tabbing through the interface in forward direction, the + button for opening a new tab gets skipped. So, after pressing the tab key when focused on the Settings button, one immediately lands on the control for switching tabs. On the other hand, moving in reverse direction with shift + tab always focuses the + button (just as it should be).
